woocommerce/woocommerce
# Center stock indicator in Product Collection list

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds the code to center the `Stock Indicator` inside the `Product Collection` block.

| Before | After |
|--------|--------|
| ![Screenshot 2024-12-26 at 3 59 02 PM](https://github.com/user-attachments/assets/f6041912-4d70-4966-9d07-9b2f0f964938) | ![Screenshot 2024-12-26 at 3 57 44 PM](https://github.com/user-attachments/assets/dcca2a18-ab80-41bf-b32b-2c41d9bbdf9a) |

Closes #53757.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a few products, some of them `Out of stock`.
2. Go to Pages > Add New Page
3. Add a `Product Collection` block and add a stock indicator in the Product Template.
4. Save the page.

https://github.com/user-attachments/assets/9e100db4-15b7-450c-8342-4243c6cf9602

5. Verify that the `Stock Indicator` is centered on the editor and the front end.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
